docs/types/response.md

```diff
@@ -3,42 +3,110 @@
 
 @signature `response(status, body, headers, statusText)`
```

Comment:
We should show what's optional with either multiple signatures or stuff like `response([status], body [,headers ][, statusText ])`

Reply:
Resolved in 162c8fa